idunn/blocks/covid19.py

```diff
@@ -0,0 +1,40 @@
+from idunn import settings
+from .base import BaseBlock
+from .opening_hour import OpeningHourBlock
+from pydantic import constr
+from typing import ClassVar, List, Optional
+
+
+class Covid19Block(BaseBlock):
+    BLOCK_TYPE: ClassVar = "covid19"
+    STATUSES: ClassVar = ["open_as_usual", "open", "maybe_open", "closed", "unknown"]
+
+    status: constr(regex="({})".format("|".join(STATUSES)))
+    opening_hours: Optional[OpeningHourBlock]
+    note: Optional[str]
+
+    @classmethod
+    def from_es(cls, es_poi, lang):
+        if settings["BLOCK_COVID_ENABLED"] is not True:
+            return None
+        opening_hours = es_poi.properties.get("opening_hours:covid19")
+        status = "unknown"
+        if opening_hours == "same":
+            opening_hours = OpeningHourBlock.from_es(es_poi.properties.get("opening_hours"), lang)
+            status = "open_as_usual"
+        elif opening_hours == "off":
+            opening_hours = None
+            status = "closed"
+        elif opening_hours is not None:
+            opening_hours = OpeningHourBlock.from_es(opening_hours, lang)
+            if opening_hours is None:
+                status = "unknown"
+            elif opening_hours.status == "open":
+                status = "open"
+            elif opening_hours.status == "closed":
+                status = "closed"
```

Comment:
Here `opening_hours.status` represents the realtime status of the POI.
I think the covid19 block `status` should be `open` in both cases, meaning that the POI is not completely closed during the lockdown.

Reply:
Then what am I supposed to put in here?